### PR TITLE
Component table update

### PR DESCRIPTION
**Pull request by boxadesign**
### What is the context of this PR?
Describe what you have changed and why, link to other PRs or Issues as appropriate.

### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

File: components/table/sortable-table.js [hunk 1]
```diff
@@ -0,0 +1,239 @@
+function SortableTable(table) {
+
+    var table = table;
+    var status;
+  
+    var options = {};
+    options.statusMessage = 'Sort by %heading% (%direction%)';
```
Comment (bameyrick) [inline components/table/sortable-table.js]: Ideally this text should come from a data attribute so that this can support other languages

**Comment by boxadesign** [comment]
@bameyrick I've completed work on the js for the sortable table variant so it's ready for review again.

**Comment by RhodGriffith** [comment]
@boxadesign @armstrongb  Tables are looking good. For consistency, should we make the sortable table headers bold as well? Could also remove the underline as they're not links, what do you think?

**Comment by armstrongb** [comment]
@RhodGriffith I think the **bold** sortable `th` is on @boxadesign 's list

**Comment by boxadesign** [comment]
@armstrongb Have updated with the additional scroll shadows and scrollbar.

**Comment by armstrongb** [comment]
> @armstrongb Have updated with the additional scroll shadows and scrollbar.

... and **bold** on `th`. LGTM

**Comment by armstrongb** [comment]
@boxadesign we're awaiting DAC report section on the sortable table.

**Comment by boxadesign** [comment]
@armstrongb Have added in some `aria` based on this http://adrianroselli.com/2018/02/tables-css-display-properties-and-aria.html

Have tested with voiceover and rotor and seems to be working in a sensible way now. Possibly one to go for a re-test as it's quite small.

**Comment by armstrongb** [comment]
Thanks @boxadesign can you add the information to the docs so that people implementing them understand necessity and need for this. Thanks.

**Comment by boxadesign** [comment]
@armstrongb Are we waiting on DAC sortable test before we merge in?

**Comment by boxadesign** [comment]
DAC had one main issue with the sortable. When the sortable `th` was focused it wasn't clear what the button did. Have added "Sort by" `aria-label` before the cell value is read out to provide more information about the `button`.

**Pull request merged**